stl/inc/type_traits

```diff
@@ -1877,8 +1934,8 @@ public:
     }
 
     template <class... _Types>
-    auto operator()(_Types&&... _Args) const -> decltype(_STD invoke(get(), _STD forward<_Types>(_Args)...)) {
-        return _STD invoke(get(), _STD forward<_Types>(_Args)...);
+    auto operator()(_Types&&... _Args) const -> decltype(_STD invoke(get(), static_cast<_Types&&>(_Args)...)) {

```

Comment:
Pre-existing: since `reference_wrapper` is tightly integrated with `invoke` (and `is_nothrow_invocable` etc. now), we should strengthen its function call operator with conditional `noexcept`. I actually view this as a defect in the Standard. We should consider adding test coverage for such strengthening (note that the "functional overhaul" tests are ridiculously comprehensive and I am satisfied that they cover the rest of the changes - it's just new behavior that needs to be tested).